### PR TITLE
Creating an initial README and GitHub matter

### PR DESCRIPTION
**Pull request by iennae**
This closes Issues #1 and #3. 

I know there is no code yet, but it's always good to have some baselines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Homebrew Tap for Porter
+
+For users on MacOS, the Porter project maintains a <a href="https://brew.sh/">Homebrew</a> tap to install and keep Porter up-to-date.
```
Comment (carolynvs) [inline README.md:3]: I would hate for someone to see the readme and believe that the tap is ready to use. Let's put up a line below the title that makes it clear that this isn't done yet.

```
🚨 This isn't ready to use yet. We will update this page with the proper commands to run once our tap is complete!
```

**Review by carolynvs — CHANGES_REQUESTED** [review]
We just need to add a warning about this tap being a WIP and then this is good to go. 👍

File: README.md (hunk 1 above)
Comment (iennae) [reply README.md:3]: Updated :) Thanks for the handy message to add.

**Review by carolynvs — APPROVED** [review]
Perfect! 🌮 🚀

**Pull request merged**